roboteam_ai/src/control/ControlUtils.cpp

```diff
@@ -262,6 +271,16 @@ Vector2 ControlUtils::calculateForce(rtt::Vector2 vector, double weight, double
     return {0, 0};
 }
 
+/// Calculates the position the robot should "aim" at given the ballPos, a targetPos and the geneva state
+Vector2 ControlUtils::getGenevaAim(Vector2 ballPos, Vector2 targetPos, int genevaState) {
```

Comment:
why not make this function work with a line rather than a point it aims at? This makes more sense as it is closer to the concept you are working with here, which is also convenient when trying to for example compute if this genevaAngle actually will hit the goal or not.

Reply:
so an std::pair<Vector2,Vector2> ?

Maybe we should also make a custom class for this object as a 'mathUtil' or a class type like Vector2? We use the concept in so many places, but it is not very centralized at the moment.